src/components/color/AccessibilityInfo.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import cx from 'classnames';
+import { readability } from 'tinycolor2';
+
+import Flex from 'meetup-web-components/lib/layout/Flex';
+import FlexItem from 'meetup-web-components/lib/layout/FlexItem';
+
+import colorData from '../../utils/colorData';
+
+const rgba2rgb = (RGB_background, RGBA_colorArr) => {
```

Comment:
It took me a few reads of this to understand what it was doing; maybe rename to `toCompositedRGB` and doc the params?

Reply:
Also, it looks like the rgb string must be converted back into an array in usage below. Should this function just return an RGB array instead, as `toCompositedRGBArray`

Reply:
You could use a separate function for converting the RGB array into an `rgb()` string:

```js
const getRgbString = arr =>
	`rgb(${arr[0]},${arr[1]},${arr[2]})`;
```

Reply:
basically type conversion and color compositing should happen separately to make things easier to understand